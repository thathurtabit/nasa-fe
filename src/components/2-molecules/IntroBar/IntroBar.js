import React from 'react';
import IntroBarStyled from './IntroBar.styled';
import Intro from '../../1-atoms/Intro/Intro';
import SearchInfo from '../../1-atoms/SearchInfo/SearchInfo';

const IntroBar = () => (
  <IntroBarStyled>
    <Intro />
    <SearchInfo />
  </IntroBarStyled>
);

export default IntroBar;
