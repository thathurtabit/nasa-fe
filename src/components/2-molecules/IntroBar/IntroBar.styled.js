import styled from 'styled-components';

const IntroBarStyled = styled.section`
  margin-bottom: 50px;
  text-align: center;

  @media screen and (min-width: ${props => props.theme.breakpoints.md}) {
    display: flex;
    align-items: center;
    justify-content: center;
  }

  h1,
  p {
    margin: 0 20px 10px;

    @media screen and (min-width: ${props => props.theme.breakpoints.md}) {
      margin: 0 20px;
    }
  }
`;

export default IntroBarStyled;
