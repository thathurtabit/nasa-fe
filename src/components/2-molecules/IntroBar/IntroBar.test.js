import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import 'jest-styled-components';
import { shallowWithTheme } from '../../../utils/test_config/testHelpers';

import IntroBar from './IntroBar';

describe('IntroBar', () => {
  it('should render correctly', () => {
    const element = shallowWithTheme(<IntroBar />);
    expect(shallowToJson(element)).toMatchSnapshot();
  });
});
