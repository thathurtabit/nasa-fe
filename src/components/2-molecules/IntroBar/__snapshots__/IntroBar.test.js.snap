// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroBar should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
